fcrepo-indexing-solr/src/main/java/org/fcrepo/camel/indexing/solr/SolrRouter.java

```diff
@@ -43,6 +43,8 @@
     private static final String hasIndexingTransformation =
         "/rdf:RDF/rdf:Description/indexing:hasIndexingTransformation/text()";
 
+    private static final String RESOURCE_DELETION = "http://fedora.info/definitions/v4/event#ResourceDeletion";
```

Comment:
This also seems like a logical candidate for inclusion in: https://github.com/fcrepo4-exts/fcrepo-camel/blob/master/src/main/java/org/fcrepo/camel/JmsHeaders.java#L21